### PR TITLE
Add invoicee email and local_business_number to invoices info

### PR DESCRIPTION
**Pull request by luisaterrabrito**

### CODE REVIEW

File: apiary.apib [hunk 1]
```diff
@@ -1883,6 +1883,8 @@ Get details for a single invoice.
                     + contact (object)
                         + type: `contact` (string)
                         + id: `f88ab83c-b8ca-497b-a2c6-1ad631fe1cea` (string)
+                + email: `duivels@test.com` (string, nullable)
```
Comment (mrtus) [inline apiary.apib:1886]: Q: Should we name this to something more explicit? Which email is represented here?

**Review by mrtus — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (jdrieghe) [reply apiary.apib:1886]: Are you suggesting something like `invoicing_email`? @stivni @jenssegers ?

Comment (dcsg) [reply apiary.apib:1886]: @mrtus having it under `invoicee` I would assume that this is the email in which the invoice will be sent, is that correct? /cc @luisabritotl

**Review by dcsg — APPROVED** [review]

File: apiary.apib (hunk 1 above)
Comment (luisaterrabrito) [reply apiary.apib:1886]: Since it is inside the invoicee, I think it is exclicit it is the invoicee/customer email

Comment (jdrieghe) [reply apiary.apib:1886]: Discussed this with @PieterReel and we decided that this is specific enough, indeed

 :ok: by @luisabritotl on 01/08/2018 10:28:01

**Review by lowiebenoot — APPROVED** [review]

**Pull request merged**